cron/cron_test.go

```diff
@@ -0,0 +1,296 @@
+package cron
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/contribsys/faktory/cli"
+	"github.com/contribsys/faktory/server"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	enabledConfig = `
+	[cron_plugin]
+	  enabled = true
+	[[ cron ]]
+	  schedule = "* * * * *" # every minute
+	  [cron.job]
+	    type = "minutely_test_job"
+		retry = 5
+		queue = "test"
+		args = [1, 3]
```

Comment:
I'd like to see assertions for these args
I'd also like to see args that are dictionaries, eg
```toml
    [[cron.job.args]]
      hello = "world"
    [[cron.job.args]]
      hello = "human"
```